languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -0,0 +1,89 @@
+use options::*;
+
+#[test]
+fn test_reviving_dead_player() {
+    let level: u32 = 34;
+    let dead_player = Player {
+        health: 0,
+        mana: Some(0),
+        level,
+    };
+    let revived_player = dead_player.revive();
+    assert_eq!(revived_player.is_some(), true);
+
+    let revived_player = revived_player.unwrap();
```

Comment:
```suggestion
    let revived_player = dead_player.revive().expect("reviving a dead player must return Some(player)");
```

Shorter and gives a clearer error message.